src/sage/graphs/graph.py

```diff
@@ -4901,16 +4901,23 @@ def independent_set_of_representatives(self, family, solver=None, verbose=0,
         return repr
 
     @doc_index("Algorithmically hard stuff")
-    def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3):
+    def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3, induced=False):
         r"""
-        Return the vertices of a minor isomorphic to `H` in the current graph.
+        Return the vertices of a minor isomorphic to `H` in the current graph for induced=False
 
         We say that a graph `G` has a `H`-minor (or that it has a graph
         isomorphic to `H` as a minor), if for all `h\in H`, there exist disjoint
         sets `S_h \subseteq V(G)` such that once the vertices of each `S_h` have
         been merged to create a new graph `G'`, this new graph contains `H` as a
         subgraph.
 
+        Returns an induced minor isomorphic to `H` if it exists for induced=True 
```

Comment:
May be better to write:

`When parameter ``induced`` is ``True``, this method returns an induced minor isomorphic to `H`, if it exists.